c7n/resources/ami.py

```diff
@@ -432,8 +432,14 @@ def process_image(self, client, image):
         remove = []
         add = []
         account_regex = re.compile('\\d{12}')
-        org_regex = re.compile('arn:[a-zA-Z-]+:organizations:\\d{12}:organization/o-.*')
-        ou_regex = re.compile('arn:[a-zA-Z-]+:organizations:\\d{12}:ou/o-.*/ou-.*')
+        # https://docs.aws.amazon.com/organizations/latest/APIReference/API_Organization.html
+        org_regex = re.compile(
+            r'arn:aws:organizations::\d{12}:organization\/o-[a-z0-9]{10,32}'
```

Comment:
the question is if this is the correct partition/arn in the case of other partitions (govcloud, china, etc).

Reply:
per https://docs.aws.amazon.com/service-authorization/latest/reference/list_awsorganizations.html#awsorganizations-resources-for-iam-policies

the partition needs to be variable.

Reply:
I pushed a partition fix.